### ISSUE TITLE
[High CPU] Output byte buffer attempts writing while the consumer connection is broken

### ISSUE THREAD

**Issue by sachithKay** [wso2/product-ei#5331]
**Description:**
Even when the consumer error or stale connection exists, the outputbytebuffer attempts to execute writes leading to high cpu issues. This can lead to high cpu usage issues with the following stack trace.
```
"PassThroughMessageProcessor-14" #275 prio=5 os_prio=31 tid=0x00007f89d22de800 nid=0x29503 runnable [0x0000700019feb000]
   java.lang.Thread.State: RUNNABLE
	at java.io.FilterInputStream.read(FilterInputStream.java:107)
	at javax.activation.DataHandler.writeTo(DataHandler.java:308)
	at org.wso2.carbon.relay.ExpandingMessageFormatter.findAndWrite2OutputStream(ExpandingMessageFormatter.java:196)
	at org.wso2.carbon.relay.ExpandingMessageFormatter.writeTo(ExpandingMessageFormatter.java:97)
	at org.apache.synapse.transport.passthru.PassThroughHttpSender.submitResponse(PassThroughHttpSender.java:610)
	at org.apache.synapse.transport.passthru.PassThroughHttpSender.invoke(PassThroughHttpSender.java:282)
	at org.apache.axis2.engine.AxisEngine.send(AxisEngine.java:442)
	at org.apache.synapse.core.axis2.Axis2Sender.sendBack(Axis2Sender.java:217)
	at org.apache.synapse.core.axis2.Axis2SynapseEnvironment.send(Axis2SynapseEnvironment.java:530)
	at org.apache.synapse.mediators.builtin.SendMediator.mediate(SendMediator.java:120)
	at org.apache.synapse.mediators.AbstractListMediator.mediate(AbstractListMediator.java:109)
	at org.apache.synapse.mediators.AbstractListMediator.mediate(AbstractListMediator.java:71)
	at org.apache.synapse.mediators.base.SequenceMediator.mediate(SequenceMediator.java:158)
	at org.apache.synapse.mediators.base.SequenceMediator.mediate(SequenceMediator.java:214)
	at org.apache.synapse.mediators.AbstractListMediator.mediate(AbstractListMediator.java:109)
	at org.apache.synapse.mediators.AbstractListMediator.mediate(AbstractListMediator.java:71)
	at org.apache.synapse.mediators.base.SequenceMediator.mediate(SequenceMediator.java:158)
	at org.apache.synapse.core.axis2.Axis2SynapseEnvironment.injectMessage(Axis2SynapseEnvironment.java:336)
	at org.apache.synapse.core.axis2.SynapseCallbackReceiver.handleMessage(SynapseCallbackReceiver.java:579)
	at org.apache.synapse.core.axis2.SynapseCallbackReceiver.receive(SynapseCallbackReceiver.java:194)
	at org.apache.axis2.engine.AxisEngine.receive(AxisEngine.java:180)
	at org.apache.synapse.transport.passthru.ClientWorker.run(ClientWorker.java:263)
	at org.apache.axis2.transport.base.threads.NativeWorkerPool$1.run(NativeWorkerPool.java:172)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor

```

**Comment by dushaniw** [comment]
Fixed in WUM updated API-M 2.5 distribution.

**Comment by dushaniw** [comment]
Fixed in WUM updated API-M 2.5 distribution.
